internal/test/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "description": "runtime dependencies for tests",
   "devDependencies": {
-    "is-builtin-module": "^2.0.0",
-    "jasmine": "~2.8.0",
+    "is-builtin-module": "2.0.0",
+    "jasmine": "2.8.0",
     "node_resolve_index": "file:./node_resolve_index",
     "node_resolve_index_2": "file:./node_resolve_index_2",
     "node_resolve_index_3": "file:./node_resolve_index_3",
     "node_resolve_index_4": "file:./node_resolve_index_4",
     "node_resolve_main": "file:./node_resolve_main",
     "node_resolve_main_2": "file:./node_resolve_main_2",
     "node_resolve_nested_main": "file:./node_resolve_nested_main",
-    "rollup": "0.52.3",
-    "rollup-plugin-node-resolve": "3.0.0",
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "unidiff": "^0.0.4"
+    "rollup": "0.60.1",
+    "rollup-plugin-node-resolve": "3.3.0",
+    "rollup-plugin-sourcemaps": "0.4.2",
```

Comment:
since we have lock files, any particular reason to remove usage of ranges?

Reply:
Best practice was the thought. Took that out and made another PR to update rollup.